pyk/src/pyk/integration_tests/test_defn.py

```diff
@@ -56,65 +51,108 @@ def test_print_prove_rewrite(self):
                                 )
         # fmt: on
 
-        minimized_claim_rewrite_expected = '\n'.join(
-            [
-                '<T>',
-                '  <k>',
-                '    ( X + Y => X +Int Y )',
-                '  </k>',
-                '  <state>',
-                '    STATE',
-                '  </state>',
-                '</T>',
-            ]
+        minimized_claim_rewrite_expected = (
+            f'<T>\n'
+            f'  <k>\n'
+            f'    ( X + Y => X +Int Y )\n'
+            f'  </k>\n'
+            f'  <state>\n'
+            f'    STATE\n'
+            f'  </state>\n'
+            f'</T>'
```

Comment:
```suggestion
            '<T>\n'
            '  <k>\n'
            '    ( X + Y => X +Int Y )\n'
            '  </k>\n'
            '  <state>\n'
            '    STATE\n'
            '  </state>\n'
            '</T>'
```

Will include `flake8` again to catch these issues.

Reply:
Done.